scripts/vulners.nse

```diff
@@ -0,0 +1,214 @@
+description = [[
+For each available cpe it prints the known vulns (links to the correspondent info).
+
+Its work is pretty simple:
+- work only when some software version is identified for an open port
+- take all the known cpe for that software (from the standard nmap output)
+- ask whether some known vulns exist for that cpe
+- print that info out
+]]
+
+---
+-- @usage 
+-- nmap -sV --script vulners [--script-args mincvss=<arg_val>] <target>
+--
+-- @output
+--
+-- 53/tcp   open     domain             ISC BIND DNS
+-- | vulners:
+-- |   ISC BIND DNS:
+-- |     CVE-2012-1667        8.5        https://vulners.com/cve/CVE-2012-1667
+-- |     CVE-2002-0651        7.5        https://vulners.com/cve/CVE-2002-0651
+-- |     CVE-2002-0029        7.5        https://vulners.com/cve/CVE-2002-0029
+-- |     CVE-2015-5986        7.1        https://vulners.com/cve/CVE-2015-5986
+-- |     CVE-2010-3615        5.0        https://vulners.com/cve/CVE-2010-3615
+-- |     CVE-2006-0987        5.0        https://vulners.com/cve/CVE-2006-0987
+-- |     CVE-2014-3214        5.0        https://vulners.com/cve/CVE-2014-3214
+--
+
+author = 'gmedian AT vulners DOT com'
+license = "Same as Nmap--See https://nmap.org/book/man-legal.html"
+categories = {"vuln", "safe"}
```

Comment:
If I understand well, this script **must** also be categorized as "external".

Reply:
Got it.